sokol_gfx.h

```diff
@@ -10029,92 +10082,114 @@ _SOKOL_PRIVATE UINT _sg_d3d11_cpu_access_flags(sg_usage usg) {
+        case SG_PIXELFORMAT_RG8:                  return DXGI_FORMAT_R8G8_UNORM;
+        case SG_PIXELFORMAT_RG8SN:                return DXGI_FORMAT_R8G8_SNORM;
+        case SG_PIXELFORMAT_RG8UI:                return DXGI_FORMAT_R8G8_UINT;
+        case SG_PIXELFORMAT_RG8SI:                return DXGI_FORMAT_R8G8_SINT;
+        case SG_PIXELFORMAT_R32UI:                return DXGI_FORMAT_R32_UINT;
+        case SG_PIXELFORMAT_R32SI:                return DXGI_FORMAT_R32_SINT;
+        case SG_PIXELFORMAT_R32F:                 return DXGI_FORMAT_R32_FLOAT;
+        case SG_PIXELFORMAT_RG16:                 return DXGI_FORMAT_R16G16_UNORM;
+        case SG_PIXELFORMAT_RG16SN:               return DXGI_FORMAT_R16G16_SNORM;
+        case SG_PIXELFORMAT_RG16UI:               return DXGI_FORMAT_R16G16_UINT;
+        case SG_PIXELFORMAT_RG16SI:               return DXGI_FORMAT_R16G16_SINT;
+        case SG_PIXELFORMAT_RG16F:                return DXGI_FORMAT_R16G16_FLOAT;
+        case SG_PIXELFORMAT_RGBA8:                return DXGI_FORMAT_R8G8B8A8_UNORM;
+        case SG_PIXELFORMAT_SRGB8A8:              return DXGI_FORMAT_R8G8B8A8_UNORM_SRGB;
+        case SG_PIXELFORMAT_RGBA8SN:              return DXGI_FORMAT_R8G8B8A8_SNORM;
+        case SG_PIXELFORMAT_RGBA8UI:              return DXGI_FORMAT_R8G8B8A8_UINT;
+        case SG_PIXELFORMAT_RGBA8SI:              return DXGI_FORMAT_R8G8B8A8_SINT;
+        case SG_PIXELFORMAT_BGRA8:                return DXGI_FORMAT_B8G8R8A8_UNORM;
+        case SG_PIXELFORMAT_RGB10A2:              return DXGI_FORMAT_R10G10B10A2_UNORM;
+        case SG_PIXELFORMAT_RG11B10F:             return DXGI_FORMAT_R11G11B10_FLOAT;
+        case SG_PIXELFORMAT_RGB9E5:               return DXGI_FORMAT_R9G9B9E5_SHAREDEXP;
+        case SG_PIXELFORMAT_RG32UI:               return DXGI_FORMAT_R32G32_UINT;
+        case SG_PIXELFORMAT_RG32SI:               return DXGI_FORMAT_R32G32_SINT;
+        case SG_PIXELFORMAT_RG32F:                return DXGI_FORMAT_R32G32_FLOAT;
+        case SG_PIXELFORMAT_RGBA16:               return DXGI_FORMAT_R16G16B16A16_UNORM;
+        case SG_PIXELFORMAT_RGBA16SN:             return DXGI_FORMAT_R16G16B16A16_SNORM;
+        case SG_PIXELFORMAT_RGBA16UI:             return DXGI_FORMAT_R16G16B16A16_UINT;
+        case SG_PIXELFORMAT_RGBA16SI:             return DXGI_FORMAT_R16G16B16A16_SINT;
+        case SG_PIXELFORMAT_RGBA16F:              return DXGI_FORMAT_R16G16B16A16_FLOAT;
+        case SG_PIXELFORMAT_RGBA32UI:             return DXGI_FORMAT_R32G32B32A32_UINT;
+        case SG_PIXELFORMAT_RGBA32SI:             return DXGI_FORMAT_R32G32B32A32_SINT;
+        case SG_PIXELFORMAT_RGBA32F:              return DXGI_FORMAT_R32G32B32A32_FLOAT;
+        case SG_PIXELFORMAT_DEPTH:                return DXGI_FORMAT_R32_TYPELESS;
+        case SG_PIXELFORMAT_DEPTH_STENCIL:        return DXGI_FORMAT_R24G8_TYPELESS;
+        case SG_PIXELFORMAT_DEPTH24PLUS:          return DXGI_FORMAT_R24G8_TYPELESS;
```

Comment:
I couldn't find a DirectX format that mapped to a 24-bit depth buffer with no stencil - so `SG_PIXELFORMAT_DEPTH24PLUS` is equivalent to `SG_PIXELFORMAT_DEPTH24PLUS_STENCIL8`. Is this the correct approach, or should this format not be supported at all on DirectX?

Reply:
I don't know either but will also try to find more info on that (and also test on Windows).